lib/professions.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Professions do
+  @type professionId :: integer()
+  @type professionCategory :: String.t()
+  @type profession :: %{id: professionId(), name: String.t(), category: professionCategory()}
+  @type t :: [profession]
+
+  defp getProfessionsByProfessionId() do
+    CSVParser.parseProfessions() |> Map.new(fn p -> {p.id, p} end)
```

Comment:
The csv file containing the professions is parser each time a profession id is retrieved?